### ISSUE TITLE
PopupMenu breaks future mouse interactions

### ISSUE THREAD

**Issue by rosshadden** [godotengine/godot#73703]
### Godot version

4.0.dev (7e79aead99)

### System information

Arch Linux, GLES3, AMD RX 7900 XTX

### Issue description

After calling `popup()` on a `PopupMenu` inside a `GraphEdit`, some future mouse interactivity breaks. By this I mean you can no longer click the editor's scroll bars, or any buttons (neither pre-existing or custom).

This issues is _not_ present if the `display/window/subwindows/embed_subwindows` option is enabled. I'm not sure whether being inside a graph matters.

### Steps to reproduce

1. Click built-in `GraphEdit` buttons to verify that they work as expected (toggle grid, etc.).
2. Try dragging the `GraphEdit` scrollbars to verify that they work as expected.
3. Click my custom "some button" I added to verify that it prints "button pressed".
4. Right click, which calls `PopupMenu#popup`, and then close the popup however you wish (clicking outside, escape, etc.).
5. Repeat steps 1-3 and see that they no longer work as expected.

### Minimal reproduction project

[mvp.zip](https://github.com/godotengine/godot/files/10797237/mvp.zip)

**Comment by Sauermann** [comment]
Can you please test, if accepting the event in your MRP solves your problem?

``` GDSCRIPT
func _input(event: InputEvent) -> void:
	if not event is InputEventMouseButton: return

	var e := event as InputEventMouseButton
	match e.button_index:
		MOUSE_BUTTON_RIGHT:
			if e.is_pressed():
				menu.position = position + get_global_mouse_position()
				menu.popup()
				accept_event() # This line is new.
```

**Comment by rosshadden** [comment]
Yes! That does solve it. So is this a user error thing, is what happens intentional? Thank you very much for the quick reply.
